### PR TITLE
Update patch plugin versions in gemfile lock

### PR DESCRIPTION
**Pull request by jsvd**

### CODE REVIEW

**Comment by elasticcla** [comment]
Hi @jsvd, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in your Git [commit](https://github.com/elastic/logstash/pull/10678.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

File: Gemfile.jruby-2.5.lock.release [hunk 1]
```diff
@@ -2,12 +2,12 @@ PATH
   remote: logstash-core-plugin-api
   specs:
     logstash-core-plugin-api (2.1.16-java)
-      logstash-core (= 7.0.1)
+      logstash-core (= 7.0.1.SNAPSHOT)
```
Comment (jsvd) [inline Gemfile.jruby-2.5.lock.release]:
```suggestion
      logstash-core (= 7.0.1)
```

File: Gemfile.jruby-2.5.lock.release [hunk 2]
```diff
@@ -2,12 +2,12 @@ PATH
...
 
 PATH
   remote: logstash-core
   specs:
-    logstash-core (7.0.1-java)
+    logstash-core (7.0.1.SNAPSHOT-java)
```
Comment (jsvd) [inline Gemfile.jruby-2.5.lock.release]:
```suggestion
    logstash-core (7.0.1-java)
```

**Comment by jsvd** [comment]
jenkins test this please

**Pull request merged**